scratch-flash.iml

```diff
@@ -57,9 +63,10 @@
       <excludeFolder url="file://$MODULE_DIR$/.gradle" />
       <excludeFolder url="file://$MODULE_DIR$/build" />
     </content>
-    <orderEntry type="jdk" jdkName="flex_sdk_4.11" jdkType="Flex SDK Type (new)" />
+    <orderEntry type="jdk" jdkName="apache-flex-4.14.1" jdkType="Flex SDK Type (new)" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" exported="" name="blooddy_crypto" level="project" />
     <orderEntry type="library" exported="" name="as3corelib" level="project" />
+    <orderEntry type="library" exported="" name="grabcut" level="application" />
```

Comment:
To fix the gradle / Travis build, add this SWC to the list of "merged files" in `scratch.gradle`.
You can test with `./gradlew build -Ptarget=11.6` (for the native build) and `./gradlew build -Ptarget=10.2` (for the pure-AIR "Linux" build).